include/prometheus/handler.h

```diff
@@ -3,10 +3,10 @@
 #include <memory>
 #include <vector>
 
-#include "CivetServer.h"
+#include <prometheus/collectable.h>
```

Comment:
Could you include your own headers with either `"prometheus/collectable.h"` or just a plain `"collectable.h"`.